rust/feature-flags/src/flags/flag_request.rs

```diff
@@ -24,11 +24,8 @@ where
     }))
 }
 
-#[derive(Debug, Clone, Copy)]
-pub enum FlagRequestType {
-    Decide,
-    LocalEvaluation,
-}
+// FlagRequestType moved to flag-analytics common crate
+pub use flag_analytics::FlagRequestType;
```

Comment:
ah, I see where the rationale of sharing this dependency across crates becomes relevant (since this service will interact with local eval billing to some degree).  Is the idea that `flag_definitions` is going to be a separate crate from `feature-flags`?  If so, I can see the case for making `flag_analytics` common, but I'd still make the case that it should only be common to flag-specific stuff (unless like the `http-server` example, that makes more sense as a common library).  What do you think?